decidim-admin/app/views/decidim/admin/categories/index.html.erb

```diff
@@ -52,7 +52,13 @@
                   <% end %>
 
                   <% if allowed_to? :destroy, :category, category: subcategory %>
-                    <%= icon_link_to "circle-x", category_path(current_participatory_space, subcategory), t("actions.destroy", scope: "decidim.admin"), class: "action-icon--remove", method: :delete, data: { confirm: t("actions.confirm_destroy", scope: "decidim.admin") } %>
+                    <% if subcategory.unused? %>
+                      <%= icon_link_to "circle-x", category_path(current_participatory_space, subcategory), t("actions.destroy", scope: "decidim.admin"), class: "action-icon--remove", method: :delete, data: { confirm: t("actions.confirm_destroy", scope: "decidim.admin") } %>
+                    <% end %>
+                  <% else %>
+                    <span class="action-icon" title="<%= t("categories.index.category_used", scope: "decidim.admin") %>" data-tooltip="true" data-disable-hover="false">
+                      <%= icon "delete-bin-line", class: "action-icon action-icon--disabled", role: "img", "aria-hidden": true %>
+                    </span>
```

Comment:
Mainly the structure needs to be: 
```
if allowed to delete 
  if there are unused sub categories 
    display link + icon 
  else 
    display icon 
  end 
end 
```
the change proposed by you, is a little bit different 

```
if allowed to delete 
  if there are unused sub categories 
    display link + icon 
  end 
else
  display icon 
end 
```



```suggestion
                    <% if subcategory.unused? %>
                      <%= icon_link_to "circle-x", category_path(current_participatory_space, subcategory), t("actions.destroy", scope: "decidim.admin"), class: "action-icon--remove", method: :delete, data: { confirm: t("actions.confirm_destroy", scope: "decidim.admin") } %>
                    <% else %>
                      <span class="action-icon" title="<%= t("categories.index.category_used", scope: "decidim.admin") %>" data-tooltip="true" data-disable-hover="false">
                        <%= icon "delete-bin-line", class: "action-icon action-icon--disabled", role: "img", "aria-hidden": true %>
                      </span>
                    <% end %>
```